### ISSUE TITLE
make it happen!

### ISSUE THREAD

**Issue by DreadKnight** [OpenKudos/javascript-client#1]
Is this a thing? :p 
I might switch over to Gitcoin soon and will get into Kudos as well big time (SVG drawings ftw!).

If this lib happens, will see about implementing it eventually in my free open source game project
https://github.com/FreezingMoon/AncientBeast
Thinking about making kudos for all its creatures, hopefully funding their development (design/art/coding/sound/animation)
https://ancientbeast.com/documentation/?id=units#focus

**Comment by owocki** [comment]
very cool :)

i should pull some numbers on how many kudos you can expect to sell before you commit to us. there ahve been about 14k kudos sent in the lifetime of gitcoin kudos

**Comment by DreadKnight** [comment]
Yeah, was thinking that the numbers might not be astronomical and it's more of a fun thing. 
Anyway, will give it a shot pretty soon. Hope the lib will start happening soon, it should get kudos some traction.
